docs/_sources/installation.rst.txt

```diff
@@ -42,7 +42,17 @@ Should be as easy as::
   git clone --recursive git@github.com:Pressio/pressio-demoapps.git
   export CXX=<path-to-your-CXX-compiler> #must support C++14
   cd pressio-demoapps
-  python setup.py install
+  # Build `pressiodemoapps` library
+  # Below e.g. (default OpenMP=OFF and build_mode=Release)
+  python scripts/cmake_build.py
+  # Below e.g. OpenMP=ON and build_mode=Debug
+  python scripts/cmake_build.py --openmp --build_mode=Debug
+  # Install dependencies
+  pip install build pytest numpy scipy matplotlib
+  # Build and install package pressiodemoapps
+  pip install .
+  # Just build a package
+  python -m build
```

Comment:
this part is not very clear to me. can you please write it in the following way: 

1. To build/install pressiodemoapps with default options:
 ... whatever I need to do 
 
2.  To build/install pressiodemoapps with openmp enabled:
 ... whatever needs to be done